third-party/llvm/Makefile.include-llvm

```diff
@@ -15,7 +15,12 @@ ifndef LLVM_LLVM_LIBS
   export LLVM_LLVM_LIBS=$(shell test -x $(LLVM_CONFIG) && $(LLVM_CONFIG) --ldflags --system-libs --libs bitreader bitwriter ipo instrumentation option objcarcopts profiledata all-targets coverage coroutines lto)
 endif
 
-LLVM_CLANG_LIBS=-lclangFrontend -lclangSerialization -lclangDriver -lclangCodeGen -lclangParse -lclangSema -lclangAnalysis -lclangEdit -lclangAST -lclangLex -lclangBasic
+ifeq ($(LLVM_MAJOR_VERSION),9)
+  # -lclangASTMatchers needed for LLVM 9 but not earlier
```

Comment:
Since the LLVM trunk is the beginning of version 10, should we be linking in this library for LLVM 9 and greater, instead of just LLVM 9 exclusively?

Reply:
That wasn't true at the time I made this PR.... Do you know offhand how to do such a numeric comparison in a Makefile snippet? (Or do we check if LLVM_MAJOR_VERSION is in a list like  9 10 ?)

Reply:
How about something like this?
```
LLVMge9 := $(shell test $(LLVM_MAJOR_VERSION) -ge 9)
ifeq ($(LLVMge9),)
 . . .
```